src/main/java/edu/hm/hafner/analysis/IssueDifference.java

```diff
@@ -31,6 +36,14 @@ public IssueDifference(final Report currentIssues, final String referenceId, fin
         fixedIssues = referenceIssues.copy();
         outstandingIssues = new Report();
 
+        referencesByHash = new HashMap();
+        referencesByFingerprint = new HashMap();
+
+        for (Issue issue : fixedIssues) {
```

Comment:
More readable:
```suggestion
        for (Issue issue : referenceIssues) {
```

Reply:
This is not the same and would case regression test failures.

Reply:
But they are copied above